src/Opdex.Platform.Infrastructure.Abstractions/Data/Queries/Tokens/TokensCursor.cs

```diff
@@ -158,6 +170,20 @@ public enum TokenAttributeFilter
     /// Opdex mined, staking tokens
     /// </summary>
     Staking = 3,
+
+    /// <summary>
+    /// Tokens wrapped by Interflux
+    /// </summary>
+    Interflux = 5
+}
+
+/// <summary>
+/// Selectable options for the token native chain
+/// </summary>
+public enum ChainType
+{
+    Cirrus = 0,
+    Ethereum = 1
 }
```

Comment:
Something we should come back to for all paginated endpoints but we can probably start here with this one.

Currently, these external client filter values match up with our database and internal enum values but this is an easy thing to miss and assumes the developer will use consistent values. An example of different external vs internal values is filtering token attributes, security attributes are not filterable, so if developers ordered enum values sequentially without knowing, it would create a mismatch and query for something else. 

We should create mappers for these public filter types to out internal database types so this won't be a future problem.